src/SourceCode.Clay/Sha1.cs

```diff
@@ -108,20 +124,26 @@ public Sha1(in ReadOnlySpan<byte> span)
         public static Sha1 Hash(in string value)
         {
             if (value == null) return Zero;
```

Comment:
There exists a document that hashes to `0000000000000000000` and it is not null. Remove the `Zero` field and throw an `ArgumentNullException` here. Null `Sha1` values can be represented by `default(Sha1?)`.

Reply:
That is a design change, I will tackle it in a separate PR

Reply:
Done